IOMC/RandomEngine/src/TRandomAdaptor.h

```diff
@@ -23,45 +23,45 @@ namespace edm {
     virtual ~TRandomAdaptor();
 
     // Returns a pseudo random number in ]0,1[ (i. e., excluding the end points).
-    double flat() { return trand_->Rndm(); }
+    double flat() override { return trand_->Rndm(); }
```

Comment:
These changes do not seem to have anything to do with changes to value_ptr. Next time please add as a seperate commit explaining the change.